src/RestaurantDetail.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import RestaurantDetail from './RestaurantDetail';
+
+import RESTAURANT from '../fixtures/restaurant';
+
+test('RestaurantDetail', () => {
+  const { getByText } = render((
+    <RestaurantDetail restaurant={RESTAURANT} />
+  ));
+
+  expect(getByText('양천주가')).not.toBeNull();
+  expect(getByText('서울 강남구')).not.toBeNull();
+  expect(getByText('탕수육')).not.toBeNull();
+  expect(getByText('팔보채')).not.toBeNull();
```

Comment:
```suggestion
  expect(container).toHaveTextContent('양천주가');
  ...
```

DOM element를 꼭 얻어야 할 상황이 아니라면 위와 같이 표현하는 게 좀 더 표현력이 좋다고 봅니다.